config.json

```diff
@@ -87,6 +87,15 @@
       ]
     },
     {
+      "uuid": "13154065-a5dc-4824-92ed-b2fae41c4dd6",
+      "slug": "protein-translation",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
```

Comment:
I was going to ask about difficulty 1 because I didn't think this exercise is difficulty 1, but given that run-length-encoding is also difficulty 1, I guess I can't complain.